docs/source/traits_user_manual/defining.rst

```diff
@@ -405,6 +405,57 @@ the table.
 |                  | *allow_none* = False, *adapt* = 'yes', \*\*\ *metadata*])|
 +------------------+----------------------------------------------------------+
 
+.. index:: Instance trait
+
+.. _instance:
+
+Instance
+::::::::
+One of the most fundamental and useful predefined trait types is Instance.
+Instance trait values are an instance of a particular class or its subclasses,
+as specified by the **klass** argument. **klass** can be either an instance of
+a class or a class itself (note this applies to all python classes, not
+necessarily just ``HasTraits`` subclasses). Further, **klass** can also be an
+interface, in which case the trait only accepts values implementing that
+interface. For more details on this particular usecase, see :ref:`interfaces`.
+
+If **klass** is an instance or if it is a class and **args** and **kw** are not
+specified, the default value is None. Otherwise, the default value is
+obtained by calling the callable **factory** argument (or **klass** if
+**factory** is None) with **args** and **kw**. For example::
+
+    # instance_trait_defaults.py --- Example of Instance trait default values
+    from traits.api import HasTraits, Instance
+
+    class Parent(HasTraits):
+        pass
+
+    class Child(HasTraits):
+        # default value is None
+        father = Instance(Parent())
+        # default value is also None
+        grandfather = Instance(Parent)
+        # default value is Parent()
+        mother = Instance(Parent, args=())
```

Comment:
Maybe this should be changed to better mirror this format, as if it were being typed in a python shell?

<img width="810" alt="Screen Shot 2021-01-14 at 12 38 27 PM" src="https://user-images.githubusercontent.com/36972686/104633859-68827d80-5665-11eb-95d4-2f576f81966a.png">

Reply:
There is also #1274 open currently.  I sort of blindly followed the existing pattern by creating a instance_trait_defaults.py and copying code here, but maybe a literal include is better.  If doing the python shell style though then thats different.

Reply:
I think I find it clearer with the class definitions as they are (so not in an interactive context). But you can always mix and match: show the class definitions in one block, and then follow up with something that looks like an interactive session using those class definitions.